localization/es/adapter/README.md

```diff
@@ -0,0 +1,139 @@
+---
+layout: pattern
+title: Adapter
+folder: adapter
+permalink: /patterns/adapter/
+categories: Structural
+language: es
+tags:
+ - Gang of Four
+---
```

Comment:
Please check the new yaml frontmatter requirements at https://github.com/iluwatar/java-design-patterns/wiki/01.-How-to-contribute